apps/funding-service/src/entry-server/middleware/index.ts

```diff
@@ -34,11 +34,16 @@ export const tokenIsValid =
       return res.status(401).json({ errors: "Access Token expired" });
     }
 
+    if (requestFunds && req.body.amount === undefined) {
+      log.verbose("missing funding amount");
+      return res.status(400).json({ errors: "Missing funding amount" });
+    }
```

Comment:
amount being inside the body is part of the validation done in express server [here](https://github.com/Rpc-h/RPCh/blob/bccebb35436508d7b8e18164a431ec4dd8668a91/apps/funding-service/src/entry-server/index.ts#L52)